plugins/db-mongo/src/index.ts

```diff
@@ -0,0 +1,82 @@
+import { resolve } from "path";
+import {
+  dataSource,
+  envVariables,
+  updateDockerComposeProperties,
+} from "./constants";
+import {
+  DsgContext,
+  AmplicationPlugin,
+  CreateServerDockerComposeDBParams,
+  CreateServerDockerComposeParams,
+  CreateServerDotEnvParams,
+  CreatePrismaSchemaParams,
+  Events,
+} from "@amplication/code-gen-types";
+
+class PostgresPlugin implements AmplicationPlugin {
+  register(): Events {
+    return {
+      CreateServerDotEnv: {
+        before: this.beforeCreateServerDotEnv,
+      },
+      CreateServerDockerCompose: {
+        before: this.beforeCreateServerDockerCompose,
+      },
+      CreateServerDockerComposeDB: {
+        before: this.beforeCreateServerDockerComposeDB,
+        after: this.afterCreateServerDockerComposeDB,
+      },
+      CreatePrismaSchema: {
+        before: this.beforeCreatePrismaSchema,
+      },
+    };
+  }
+
+  beforeCreateServerDotEnv(
+    context: DsgContext,
+    eventParams: CreateServerDotEnvParams
+  ) {
+    eventParams.envVariables = [...eventParams.envVariables, ...envVariables];
+
+    return eventParams;
+  }
+
+  beforeCreateServerDockerCompose(
```

Comment:
On MySQL plugin we had to change some variables (db-user, db-password, db-port).
Did you run the app with the current environment variables as-is? (the default Postgres values)
```
npm run docker:db
npm run prisma generate
npm run db:init
```

In general, I think that each db-plugin should override these variables to the expected values, for example:
Postgres: 5432 (the default)
MySQL: 3306 (the plugin overrides the default)
MongoDB: 27017 (up to you)
@yuval-hazaz, what do you think?

Reply:
We expose these parameters in the UI so the user may override the values for their needs. the internal port on the docker is correct (27017)
I am OK with proceeding the way it is, and we will review it soon as part of the bigger pictures on how we install DB plugins from the UI and wether we keep the DB settings page and more specifically the port parameter